### PR TITLE
overlayRef.attached is not a condition to prevent closePopover

### PR DESCRIPTION
**Pull request by guillerot**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1856/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1856/storybook

**Review by shajz — DISMISSED** [review]

File: packages/ng/popover/src/lib/trigger/popover-trigger.model.ts [hunk 1]
```diff
@@ -199,8 +199,10 @@ export abstract class ALuPopoverTrigger<TPanel extends ILuPopoverPanel = ILuPopo
 
 	/** Closes the popover. */
 	closePopover(): void {
-		if (this._overlayRef && this._overlayRef.hasAttached()) {
-			this._overlayRef.detach();
+		if (this._overlayRef) {
```
Comment (GuillaumeNury) [inline packages/ng/popover/src/lib/trigger/popover-trigger.model.ts:202]: should we call `emitClose` when we have an unattached overlayRef ?

Comment (GuillaumeNury) [reply packages/ng/popover/src/lib/trigger/popover-trigger.model.ts:202]: > Maybe there is a way to call emitClose sooner (when the "real" close occurs with the scrolling)

I would prefer that.

In the `_createOverlay` method, we could use something like `this._overlayRef.detachments().subscribe(...)` to detect popover closing.
But with this, we have to make sure that close event is not emitted twice in the "normal" use-case.

Comment (guillerot) [reply packages/ng/popover/src/lib/trigger/popover-trigger.model.ts:213]: @GuillaumeNury i've tried to take advantage of `_overlayRef.detachments`
See my last proposal

**Review by GuillaumeNury — APPROVED** [review]
Thx @guillerot

**Review by shajz — APPROVED** [review]
